sway-lib-std/src/tx.sw

```diff
@@ -232,6 +233,29 @@ pub fn tx_script_bytecode<T>() -> T {
     read::<T>(tx_script_start_pointer())
 }
 
+/// Get the hash of the script bytecode.
+/// Reverts if not a transaction-script
+pub fn tx_script_bytecode_hash() -> b256 {
+    let type = tx_type();
+    match type {
```

Comment:
```suggestion
    match tx_type() {
```

Since the variable `type` is unused and doesn't cause indentation it is cleaner to include it in the `match` directly

Reply:
I have a feeling this doesn't work but I could be wrong - I hope I'm wrong. If not, please file an issue @pixelcircuits

Reply:
I don't think it works. Every function in the library does it this way (which I copy-pasted from). I'll test to verify and then file an issue if I can confirm it

Reply:
So it works! Whatever issue there was before, it seems to have been fixed now

Reply:
Great! Glad to see these cleaned up.